src/components/input/range/style.css

```diff
@@ -39,47 +56,37 @@ input[type=range] {
 /* WebKit/Blink */
 input[type=range]::-webkit-slider-thumb {
   -webkit-appearance: none;
-  height: 2em;
-  width: 2em;
-  border-radius: 50%;
+  height: 1.5em;
+  width: .8em;
+  border-radius: 20%;
   background-color: rgb(var(--smoothly-primary-contrast));
-	border: 1px rgb(var(--smoothly-primary-contrast)) solid;
+	border: 1px #333 solid;
   cursor: pointer;
-  margin-top: .5em; /* You need to specify a margin in Chrome, but in Firefox and IE it is automatic */
+  margin-top: -.5em; /* You need to specify a margin in Chrome, but in Firefox and IE it is automatic */
 }
 
 /* Firefox */
 input[type=range]::-moz-range-thumb {
-  height: 2em;
-  width: 2em;
-  border-radius: 50%;
+  height: 1.5em;
+  width: .8em;
+  border-radius: 20%;
   background-color: rgb(var(--smoothly-primary-contrast));
-	border: 1px rgb(var(--smoothly-primary-contrast)) solid;
+	border: 1px #333 solid;
```

Comment:
do not use your own colors. use the smoothly colors. The color you have defined here might work for your current theme but you never know what someone elses theme will look like.